src/status_im2/contexts/quo_preview/settings/accounts.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.quo-preview.settings.accounts
+  (:require [clojure.string :as string]
+            [quo2.components.settings.accounts.view :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as r]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Custom color"
+    :key     :custom-color
+    :type    :select
+    :options (mapv (fn [[k _]]
```

Comment:
Unless there's a strict need for the collection to be a vector, we prefer to keep it as a lazy sequence, so just use `map`.

Reply:
OK! Solved in 496d54c1fd7d4d80b320f45666e39f5682e90a14